manifests/m/Max/Backup/0.1.3/Max.Backup.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Max.Backup

```

Hi @kellyelton,

The PackageIdentifier should be in form `Author.Package`: 

```suggestion
PackageIdentifier: kellyelton.MaxBackup
```

Does this look okay to you? 
Please remember to update the file names and paths, and other PackageIdentifiers, to match.